addons/sale/data/mail_message_subtype_data.xml

```diff
@@ -8,6 +8,12 @@
             <field name="default" eval="False"/>
             <field name="description">Quotation sent</field>
         </record>
+        <record id="mt_order_viewed" model="mail.message.subtype">
+            <field name="name">Quotation Viewed</field>
+            <field name="res_model">sale.order</field>
+            <field name="default" eval="True"/>
+            <field name="description">Quotation Viewed</field>
+        </record>
```

Comment:
I think sequences are missing for those subtypes too to enforce the order requested by DALA
![image](https://github.com/user-attachments/assets/5c34306d-43f4-4978-a103-d385dc09bf34)

Reply:
Strange that I had a difference locally.
But anyway, if an existing database upgrades, the id of this subtype will be greater than the existing ones, so I believe that the order won't be guaranteed.

Reply:
Even if we add a sequence field for these subtypes, the order will not change because 'Quotation Viewed' is an internal subtype, so it will always appear after the other two subtypes. This behavior is due to the following controller:[https://github.com/odoo/odoo/blob/d0c3e3902420b0c0fa9c3b7e0c915f08cb865147/addons/mail/controllers/thread.py#L44C4-L74C10](link)

Reply:
Oh, good investigation :+1:
@dalaOdoo is it a problem if the `Quotation Viewed` checkbox is last in the choice?  Doesn't seem critical to me that it's not between Quote sent and confirmed, since the SO can be accessed at any time by the customer anyway.

Reply:
no big deal